src/mixins/data_mutators.js

```diff
@@ -11,6 +11,7 @@ export default {
 
     genericMutator(value) {
       // if (_.isNil(value)) return NULL
+      if(value instanceof Buffer) return value.toString()
```

Comment:
or `_.isBuffer(value)`?

Reply:
If you wish i can use that, but it would create a unnecessary overhead because its would check the same thing.

Reply:
doesn't matter to me either way honestly